pyam/iiasa.py

```diff
@@ -354,9 +364,9 @@ def read_iiasa(name, meta=False, **kwargs):
     kwargs :
         Arguments for pyam.iiasa.Connection.query()
     """
-    conn = Connection(name)
+    conn = Connection(name, creds)
     # data
-    df = conn.query(**kwargs)
+    df = conn.query(creds=creds, **kwargs)
```

Comment:
if the connection already has creds, why are they needed here as well?

Reply:
I think that's the artifact separating creds from read_iiasa kwargs (just passing all the arguments as before). Will remove